apm-sniffer/apm-sdk-plugin/jdbc-commons/src/main/java/org/apache/skywalking/apm/plugin/jdbc/connectionurl/parser/ImpalaJdbcURLParser.java

```diff
@@ -99,7 +102,7 @@ public ConnectionInfo parse() {
         } else {
             String[] hostAndPort = hostSegment[0].split(":");
             if (hostAndPort.length != 1) {
-                return new ConnectionInfo(component, DB_TYPE, hostAndPort[0], Integer.valueOf(hostAndPort[1]), fetchDatabaseNameFromURL(location
+                return new ConnectionInfo(component, DB_TYPE, hostAndPort[0], Integer.parseInt(hostAndPort[1]), fetchDatabaseNameFromURL(location
```

Comment:
Could you explain why change this method?

Reply:
The ConnectionInfo's param `port` need type int not Integer, whether `Integer.parseInt` is better?

Reply:
My question is why do you use parseInt insteads of valueOf.

Reply:
I see the ConnectionInfo's port need type `int`, so `Integer.parseInt` is enough, and `Integer.valueOf`(Integer.valueOf(parseInt()) here return `Integer` may be unnecessary.

Reply:
Revert `Integer .valueOf` changes, and fix impala url parse exception only first.